onboarding/src/Components/LeftMenu.js

```diff
@@ -95,6 +98,12 @@ function LeftMenu(props) {
                   {link}
                 </li>
               ))}
+              <li className="menu-item">
+                <NavLink to="/q_and_a" className="menu-link" activeStyle={{color: "#346CB0"}}>
```

Comment:
Same reason as for all examples above.
Solutions:
`{{ pathname: "/q_and_a", state: { loggedUser: loggedUser } }}`
or even (if "q_and_a" page is expected to keep current package)
`{{ pathname: "/q_and_a", state: { packageId: packageIdRef.current, loggedUser: loggedUser } }}`
(*)
```suggestion
                <NavLink to={{ pathname: "/q_and_a", state: { packageId: packageIdRef.current, loggedUser: loggedUser } }} className="menu-link" activeStyle={{color: "#346CB0"}}>
```